csv2docx/csv2docx.py

```diff
@@ -1,12 +1,44 @@
 import csv
 from mailmerge import MailMerge
-def convert(data, template, delimiter=";"):
+from pathlib import Path
+
+def create_path_if_not_exists(path: str):
+    """Creates a path to store output data if it does not exists.
+    Args:
+        path: the path from user in any format (relative, absolute, etc.)
+    Returns:
+        A path to store output data.
+    """
+    path = Path(path)
+    if not path.exists():
+        path.mkdir(parents=True, exist_ok=True)
+    return path
+
+def generate_name(filename, path):
+    """Checks whether file with same name exists or not
+    Args:
+        filename: the file we want to check
+        path: the directory we want to check in
+    Returns:
+        An unique full path with directory.
+    """
+    filename += ".docx"
+    checkpath = path / filename
```

Comment:
can we rename `checkpath` to `filepath` because this method returns the path of the file to create